packages/components/src/__future__/Workflow/_docs/Workflow.mdx

```diff
@@ -0,0 +1,172 @@
+import { ArgTypes, Canvas, Meta } from "@storybook/blocks"
+import { Installation, KaioNotification, ResourceLinks,LinkTo } from "../../../../../../storybook/components"
+import { InlineNotification } from "@kaizen/notification"
+import * as Workflow from "./Workflow.stories"
+import * as WorkflowFooter from "./WorkflowFooter.stories"
+import * as WorkflowHeader from "./WorkflowHeader.stories"
+
+<Meta title="Pages/Workflow" />
+
+# Workflow
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Workflow"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<InlineNotification type="cautionary" persistent>
+  {`This is the latest version of the Workflow component. If you are using "@kaizen/components" at 1.16.0 or lower, please refer to our legacy documentation.`} 
```

Comment:
```suggestion
  {`If you are using "@kaizen/components" at 1.16.0 or lower, please refer to our `} <LinkTo pageId="pages-workflow-legacy">legacy documentation.</ LinkTo>
```
Everything is always "the latest version" 😉

Reply:
Also don't forget to import:
```
import { LinkTo } from "../../storybook/components/LinkTo"
```

Reply:
Actually had issues with the MDX renderer when I tried to use LinkTo as a child InlineNotification within MDX
![Screenshot 2023-06-20 at 5 03 52 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/31211daf-9ce0-4c10-b9a4-46faa6da13d0)
I also tried just adding in anchor tag, but doesn't work too well either since the styles are inconsistent.

Reply:
That said looking at the KaioNotification it looks fine if you create the component outside of the MDX

Reply:
Oh interesting, never mind then. I'll make a ticket to create a component.

Reply:
Yeah its an odd one, I assume the MDX parser has some issues Storybook's LinkTo component. I also tried the function `linkTo` to see if I could attach it to a button, but didn't have much luck with that either.